### ISSUE TITLE
Font Options, 

### ISSUE THREAD

**Issue by iAmChump** [HatchesPls/GrandTheftAutoV-Cheat#17]
Hey brotha, been a little bit hope all is well with you , so im a bit of a noob when it comes to cpp I mean im SUPER NOOB ahahha, i got some skills in other things, such as Ui. but doing what u do my friend, well lets just say I cant even dine at the same table as u , ur on a next level ahhaa, but i do have a challenge for you that ive been attempting lol maybe youll be able to do what i cant hahah lol but i would love to see it .. heres what ive been attempting with no success hahaha 

https://i.imgur.com/PrR8Hva.png 
https://i.imgur.com/LFxRx1x.png
https://i.imgur.com/Mxk0EtD.png

Basically trying to make it grab a .ttf file font, an have it switch between those, 
but ill be honest ive had no luck . crash nuff times hahaha, im to new for this, was wondering maybe u can shed some light or even flip a switch an make it work ahhahaha, but this would be hellllla cool if we could figure his out, ill keep trying but , shit i cant see why it wont work

**Comment by HatchesPls** [comment]
As far as I know you cannot make RAGE (Rockstar game engine) use any external fonts (at least not with the available render functions). To realize this you would need to render the UI yourself using DirectX directly or with a library such as [Dear ImGui](https://github.com/ocornut/imgui) (like I have done with SMI [here](https://github.com/HatchesPls/SimpleModuleInjector/blob/v1.1.0.1/Source/Main.cpp#L30)).

**Comment by iAmChump** [comment]
also , im a ui designer , if u ever want any sexy themes or anything made for your textures :D im your guy 

https://i.imgur.com/VMIz280.png
https://i.imgur.com/S5ZTQnS.png
https://i.imgur.com/s0lKwiH.png

**Comment by HatchesPls** [comment]
> So if my Base im using has IMGUI included , is this something i may be able to add in ?
Yes but you would need to completely redo the entire UI, building it from the ground up. Just for the purpose of allowing custom fonts wouldn't seem the best decision time-wise, definitely fun though.

I have considered adding ImGui for it's intended purpose though; replacing the Win32 console window that is spawned now and rendering debug information within the game window itself. I'm sure this is something I will revisit in the future once I'm done with the things I have planned right now.

> also , im a ui designer , if u ever want any sexy themes or anything made for your textures :D im your guy
> 
> https://i.imgur.com/VMIz280.png https://i.imgur.com/S5ZTQnS.png https://i.imgur.com/s0lKwiH.png
Good to know!
